bridges/TwitterBridge.php

```diff
@@ -239,7 +239,13 @@ public function collectData(){
 
 			// generate the title
 			$item['title'] = $tweet->full_text;
-			$cleanedTweet  = $tweet->full_text;
+			$cleanedTweet = $tweet->full_text;
+			$reg_ex = '/(http|https|ftp|ftps)\:\/\/[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,3}(\/\S*)?/';
+			if(preg_match($reg_ex, $tweet->full_text, $url)) {
+				$cleanedTweet = preg_replace($reg_ex, "<a href='{$url[0]}' target='_blank'>{$url[0]}</a> ", $tweet->full_text);
+			} else {
+				$cleanedTweet = $tweet->full_text;
+			}
```

Comment:
Don't think we need to run `$cleanedTweet = $tweet->full_text;` in `else` body, 'cos it is already same before running `if` statement.

Reply:
Arg, my bad. The first $cleanedTweet = $tweet->full_text; is not supposed to be here

Reply:
fixed in latest commit